requirements/requirements.txt

```diff
@@ -23,14 +21,11 @@ ovos_workshop~=0.0, >=0.0.12a4
 ovos_PHAL~=0.0, >=0.0.2
 ovos-lingua-franca>=0.4.6
 
-ovos-stt-plugin-server~=0.0, >=0.0.2
+ovos-listener~=0.0, >=0.0.2a9
```

Comment:
There is no `ovos_listener` package included here when I install locally. It seems `setup.py` is missing a `packages` spec https://github.com/OpenVoiceOS/ovos-listener/blob/dev/setup.py#L66